PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html

```diff
@@ -1,973 +1,997 @@
+            </th>
+            <th class="col-stat stat-attack">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Atk, 1)" 
+                    (click)="sort(column.Atk, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">ATK</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Atk, -1)" 
+                    (click)="sort(column.Atk, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-defense">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Def, 1)" 
+                    (click)="sort(column.Def, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">DEF</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Def, -1)" 
+                    (click)="sort(column.Def, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-special-attack">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spa, 1)" 
+                    (click)="sort(column.Spa, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPA</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spa, -1)" 
+                    (click)="sort(column.Spa, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-special-defense">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spd, 1)" 
+                    (click)="sort(column.Spd, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPD</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spd, -1)" 
+                    (click)="sort(column.Spd, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-speed">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spe, 1)" 
+                    (click)="sort(column.Spe, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPE</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spe, -1)" 
+                    (click)="sort(column.Spe, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-bulk">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Bulk, 1)" 
+                    (click)="sort(column.Bulk, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="has-tooltip">
+                    <div class="table-header-label">Bulk</div>

```

Comment:
Add dotted underline to "Bulk" text to show that there is a tooltip on this text.
CSS:
text-decoration: underline;
text-decoration-style: dotted;
text-decoration-color: var(--light-gray);

Reply:
has been added to css now